sites/example-project/src/components/viz/ErrorChart.svelte

```diff
@@ -3,62 +3,13 @@
     export let chartType;
 </script>
 
-<div width=100% class="error">
-    <div class="wrapper">
-        <div class="chart-type">{chartType}</div>
+<div class="grid [grid-template-rows:auto] grid-cols-[100%] justify-center bg-red-50 text-grey-700 font-ui font-normal rounded border border-red-100 min-h-[150px] box-content py-5 px-[30px] my-5 print:break-inside-avoid">
+    <div class="m-auto w-full">
+        <div class="font-bold text-center text-[1.2rem]">{chartType}</div>
```

Comment:
@nidhi-kala I know you're doing a 1:1 mapping of what we already have here, which is great, but I'd be interested in your thoughts: 

Could we get _close_ to what we already have by using the existing tailwind font scales (e.g. `sm`, `md` etc.)? Or would would we need to add some additional scales to our tailwind config to get close to what we're already doing?

Reply:
Closing because this is resolved/updated.